js/ui/map.js

```diff
@@ -405,10 +405,20 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
      * which are fully evaluated for the given zoom level and feature.
      *
      * @example
+     * // Find all features at a point
      * var features = map.queryRenderedFeatures([20, 35], { layers: ['my-layer-name'] });
      *
      * @example
+     * // Find all features within a static bounding box
      * var features = map.queryRenderedFeatures([[10, 20], [30, 50]], { layers: ['my-layer-name'] });
+     *
+     * @example
+     * // Find all features within a bounding box around a point
+     * var width = 10;
+     * var features = map.queryRenderedFeatures([
+     *   [point.x - width / 2, point.y - width / 2],
+     *   [point.x + width / 2, point.y + width / 2]
```

Comment:
There should be separate `width` and `height` variables, even if their values happen to be the same.